.circleci/config.yml

```diff
@@ -15,39 +15,22 @@ jobs:
             pip install \
               docker-compose==1.12.0 \
               awscli==1.11.76
-      - restore_cache:
-          keys:
-            - v1-{{ .Branch }}
-          paths:
-            - /caches/app.tar
       - run:
-          name: Load Docker image layer cache
-          command: |
-            set +o pipefail
-            docker load -i /caches/app.tar | true
-      - run:
-          name: Build application Docker image
-          command: |
-            docker build --cache-from=app -t app .
-      - run:
-          name: Save Docker image layer cache
-          command: |
-            mkdir -p /caches
-            docker save -o /caches/app.tar app
-      - save_cache:
-          key: v1-{{ .Branch }}-{{ epoch }}
-          paths:
-            - /caches/app.tar
+          name: Docker Build
+          command:
+            docker build -t refapp .
       - run:
           name: Run tests
           command: |
             docker-compose -f ./docker-compose.test.yml up
       - deploy:
           name: Push application Docker image
           command: |
-            #if [ "${CIRCLE_BRANCH}" == "master" ]; then
+             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               login="$(aws ecr get-login)"
               ${login}
-              docker tag app "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}2:${CIRCLE_PROJECT_REPONAME}${CIRCLE_SHA1}"
-              docker push "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}2:${CIRCLE_PROJECT_REPONAME}${CIRCLE_SHA1}"
-            #fi
\ No newline at end of file
+              docker tag refapp "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}:latest"
+              docker push "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}:latest"
+              docker tag refapp "${ECR_ENDPOINT}/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1}"
```

Comment:
To summarise the outcome of the discussion on what container registry structure we should use, we've decided on this:

`${ECR_ENDPOINT}/${TEAM_NAME}/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1}`

which adds `${TEAM_NAME}` into the mix, which should be the user's github team name, in `slug` format. As that will vary depending on the team used, it likely makes sense to define that as an environment variable in CircleCI.